### PR TITLE
Simple implementation of mute and unmute

### PR DESCRIPTION
**Pull request by slashformotion**
Add a simple shortcut to mute and unmute the player using the "m" key (similar to mpv)

### CODE REVIEW

**Comment by DucNg** [comment]
Nice thanks a lot for your contribution!

Just tested, it works well! Just a little suggestion, could you add a status message showing the mute status?

You can use `m.list.NewStatusMessage()`. Something similar was done for the volume: https://github.com/DucNg/fip-player/pull/8

**Comment by slashformotion** [comment]
> Nice thanks a lot for your contribution!
> 
> Just tested, it works well! Just a little suggestion, could you add a status message showing the mute status?
> 
> You can use `m.list.NewStatusMessage()`. Something similar was done for the volume: #8

done

File: player/mpv.go [hunk 1]
```diff
@@ -175,6 +176,18 @@ func (mpv *MPV) Pause() {
 	mpv.setProperty("pause", "yes")
 }
 
+func (mpv *MPV) ToggleMute() {
+	mute := "no"
+	if mpv.mute {
+		mute = "yes"
+	}
+	mpv.mute = !mpv.mute
```
Comment (DucNg) [inline player/mpv.go]: Condition is reversed without this change. You want to change the value before using it to set the property.

```suggestion
	mpv.mute = !mpv.mute
	mute := "no"
	if mpv.mute {
		mute = "yes"
	}
```

**Review by DucNg — CHANGES_REQUESTED** [review]
Condition is reversed

File: player/mpv.go (hunk 1 above)
Comment (slashformotion) [reply player/mpv.go]: done

**Review by DucNg — APPROVED** [review]
Thank you very much! :pray:

**Pull request merged**